manifests/broker/service.pp

```diff
@@ -36,31 +36,14 @@
     }
     $environment = deep_merge($env_defaults, $env)
 
-    if $facts['service_provider'] == 'systemd' {
-      include systemd
+    include systemd
 
-      file { "/etc/systemd/system/${service_name}.service":
-        ensure  => file,
-        mode    => '0644',
-        content => template('kafka/unit.erb'),
-      }
-
-      file { "/etc/init.d/${service_name}":
-        ensure => absent,
-      }
-
-      File["/etc/systemd/system/${service_name}.service"]
-      ~> Service[$service_name]
-    } else {
-      file { "/etc/init.d/${service_name}":
-        ensure  => file,
-        mode    => '0755',
-        content => template('kafka/init.erb'),
-        before  => Service[$service_name],
-      }
+    file { "/etc/systemd/system/${service_name}.service":
```

Comment:
In the future we should migrate this to systemd::unit_file

Reply:
Yes we should. But I recon that should maybe be another PR